### ISSUE TITLE
Blog Sample Data on branch 4.3-dev

### ISSUE THREAD

**Issue by astridx** [joomla/joomla-cms#39836]
### Steps to reproduce the issue

Install `blog sample data `on branch 4.3-dev

### Expected result

Sample data are installed

### Actual result

![sampledateerror](https://user-images.githubusercontent.com/9974686/218251869-17ab2cb3-463f-40f0-940c-bb4ac4acf3e7.png)

### System information (as much as possible)

I checked it. There was no tag in the trash and no active tag in the tags component.

[systeminfo-2023-02-11T10 56 23+01 00.txt](https://github.com/joomla/joomla-cms/files/10713107/systeminfo-2023-02-11T10.56.23%2B01.00.txt)

### Additional comments

After that it was possible for me to install the `testing sample data`.

**Comment by richard67** [comment]
I cannot reproduce this on a clean current 4.3-dev branch. Are you sure you did not run into your other issue with trashed items not being shown? And are you sure there was no previous attempt to install any kind of sample data? I assume you did check that, am just asking to be sure.

**Comment by alikon** [comment]
starting from 4.3.0-alpha4-dev to nightly  no issue 
@astridx did have a multilanguage setup ?

**Comment by brianteeman** [comment]
Must be related to your other issue

**Comment by astridx** [comment]
I'm sorry. I can't recreate it now either. I must have done something wrong.

The error occurred during the installation for https://github.com/joomla/joomla-cms/pull/39805, where I changed from branch 4.2-dev to 4.3-dev. 

After changing the branches, I deleted `configuration.php`, ran `npm `and `composer `and reinstalled Joomla. When I then wanted to install the blog sample data, it didn't work and the tags were reported as errors. 

I then used the test sample data for the PR. That worked. 

When I reinstalled everything, the error no longer occurred. Because the data in the PR shows the correct diff to `4.3-dev`, I assume that composer or node did not run correctly this morning and I had missed it.
